examples/advection_1d/test_advection_nonunif.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import absolute_import
+def test_1d_advection():
+    """test_1d_advection
+
+    tests against expected classic, sharpclaw, and high-order weno results """
+
+    from . import advection_1d
```

Comment:
I think this should be `import advection_1d_nonunif` (see the error log on Travis).

For consistency with names used elsewhere, it would be nice to use the name `advection_1d_mapped`, but if you prefer I can make this change later.